Controller/UserApi/DocumentController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\ClientHelperBundle\Controller\UserApi;
+
+use EMS\ClientHelperBundle\Service\UserApi\DocumentService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+final class DocumentController
+{
+    /** @var DocumentService */
+    private $service;
+
+    public function __construct(DocumentService $service)
+    {
+        $this->service = $service;
+    }
+
+    public function show(string $contentType, string $ouuid, Request $request): JsonResponse
+    {
+        $context = [
+            'authToken' => $request->headers->get('X-Auth-Token'),
+        ];
+
+        return new JsonResponse($this->service->getDocument($contentType, $ouuid, $context));
```

Comment:
```suggestion
        return new JsonResponse($this->service->getDocument($contentType, $ouuid, $request));
```
The controller is your glue, the service can extract auth information from the headers itself (make it reusable for the other functions!)